app/Http/Controllers/Process/ProcessStore.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace App\Http\Controllers\Process;
+
+use App\Http\Controllers\Controller;
+use App\Models\Process;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Auth;
+
+class ProcessStore extends Controller
+{
+    /**
+     * @param Request $request
+     * @return \Illuminate\Http\RedirectResponse
+     */
+    public function __invoke(Request $request)
+    {
+        $process = new Process($request->all());
+        $process->user_id = Auth::user()->getAuthIdentifier();
```

Comment:
```suggestion
        $process->user_id = request()->user()->getAuthIdentifier();
```

Creio que deixar o id exposto explicitamente não fica muito legal, pode ser passado o model de User inteiro.